This commit includes:

deepshikhaaa opened the pull request:
- distributed-server.yml: add a task to build glusterfs
- distributed-test-runner.py: remove steps of copy and compile to remote machines

Signed-off-by: Deepshikha Khandelwal <dkhandel@redhat.com>

nigelbabu commented on distributed-server.yml:
```diff
@@ -0,0 +1,176 @@
+    git:
+      repo: 'git://review.gluster.org/glusterfs'
+      dest: '/tmp/gluster-test/glusterfs'
+      version: HEAD
```
Might want to make this a variable so we can override this when needed.

deepshikhaaa committed:
This commit includes:

- distributed-server.yml: add a task to build glusterfs
- distributed-test-runner.py: remove steps of copy and compile to remote machines

Signed-off-by: Deepshikha Khandelwal <dkhandel@redhat.com>

deepshikhaaa committed:
Test with the new script
